packages/components/src/Header/Header.spec.js

```diff
@@ -0,0 +1,85 @@
+import { render } from '@testing-library/react';
+import { ThemeProvider, createTheme } from '@platzily-ui/styling';
+import Header from './Header';
+
+describe('@Components/Header', () => {
+  const spy = {};
+
+  beforeEach(() => {
+    spy.console = jest.spyOn(console, 'error').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    spy.console.mockRestore();
+  });
+
+  it('Given a styling by style attribute using theme with createTheme() ', () => {
```

Comment:
The utterance is partially good, you're using the Given but not the when and then :grin:. Here is some documentation about it https://martinfowler.com/bliki/GivenWhenThen.html.

The thing here is that I dont really get what are we testing? Are we testing that style prop get to the component?